### PR TITLE
Rewrite creating new recipe

### PR DESCRIPTION
**Pull request by RodPaDev**
# Description

This is a **major change** and **is untested**! Several things need to be tweaked before a final version and I'm at the disposal of the team handling the styling if any information or change is needed for the final product

Note:
Some features are not implemented because of the time limitations.

- No ability to add several tags or new tags
- No ability to add new ingredients

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change status

- [x] Complete, but not tested (may need new tests)

# How Has This Been Tested?

Please describe steps taken to ensure this feature is functional and does not break other features:
**NOT TESTED**

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, if relevant
- [ ] I have run the app using my feature and ensured that no functionality is broken
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

**Review by mpereannor — APPROVED** [review]

**Review by temitopeakinsoto — APPROVED** [review]

File: src/actions/actionCreators.js [hunk 1]
```diff
@@ -4,10 +4,8 @@ import axios from "axios";
 export const register = (credentials, history) => dispatch => {
   dispatch({ type: types.REQUEST_START });
   axios
-    .post(
-      "https://develop-forkbook.herokuapp.com/api/auth/register",
-      credentials
-    )
+    // .post("https://develop-forkbook.herokuapp.com/api/auth/register",credentials )
+    .post("http://localhost:3333/api/auth/register",credentials )
```
Comment (carnundotcom) [inline src/actions/actionCreators.js:8]: Why is this being sent to localhost? :/

Comment (carnundotcom) [reply src/actions/actionCreators.js:8]: Below, too.

File: src/components/createRecipe/createRecipe/CreateRecipe.jsx [hunk 2]
```diff
@@ -1,288 +1,33 @@
...
-        </div>
-      );
-
-    default:
-      return <p>Hello there</p>;
-  }
+  return (
+    <div className="App">
+      <h1> Create Recipe </h1>
+      {page === 1 && <Step1 goForward={goForward} />}
```
Comment (carnundotcom) [inline src/components/createRecipe/createRecipe/CreateRecipe.jsx:23]: Nice!

File: src/components/recipe/SeeRecipe.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ import {
   ProfilePicture,
   BottomButtonDiv,
   LgButton
-} from "./Recipe.styles";
+} from "./recipe.styles";
```
Comment (carnundotcom) [inline src/components/recipe/SeeRecipe.js:28]: Better to change the file name, I think! More consistent.

**Review by carnundotcom — APPROVED** [review]
Like it! Muuuuuuch neater than previously! :D

(Left a couple comments.)

**Pull request merged**